cmake/Modules/LibAddMacros.cmake

```diff
@@ -529,7 +529,7 @@ if (BUILD_DOCUMENTATION AND RONN_LOC) # disable function when RONN_LOC is not se
 	add_custom_command(
 		OUTPUT ${OUTFILE}
 		DEPENDS ${MDFILE}
-		COMMAND ${RONN_LOC}
+		COMMAND export RUBYOPT="-Eutf-8" && ${RONN_LOC}
```

Comment:
> Is maybe `RUBYOPT="-Eutf-8" ${RONN_LOC}` more portable?

I already tried that. Setting the environment variable only for the command does not work.

> Or even better would be `${CMAKE_COMMAND} -E env RUBYOPT="-Eutf-8" ${RONN_LOC}` but it is not available in cmake 3.

This works, but as far as I know only in [**CMake 3.2.**](http://public.kitware.com/Bug/view.php?id=5145#c36770) or newer. Since some of the build servers use old versions of CMake (2.x) I decided against using `cmake -E env`.